src/components/SubscriptAvatar.js

```diff
@@ -55,6 +55,7 @@ function SubscriptAvatar(props) {
             >
                 <View>
                     <Avatar
+                        containerStyles={StyleUtils.getAvatarStyle(props.size || CONST.AVATAR_SIZE.DEFAULT)}
```

Comment:
@parasharrajat yeah I agree we should replace this change with the result from this convo  https://github.com/Expensify/App/pull/21657/files#r1275373499

Reply:
I don't think this change breaks any type of subscript avatar, should we just leave it as is?

Reply:
@parasharrajat we could just remove the color style here